### ISSUE TITLE
 torchvision support `mobilenet` and `shufflenet` 

### ISSUE THREAD

**Issue by liminn** [creafz/pytorch-cnn-finetune#14]
Hi, thanks for sharing your great work!
I found that the torchvision has been updated, and it support `mobilenet`, `shufflenet` and so on. 
Will you update your repository?

**Comment by creafz** [comment]
Hi @liminn. 

Yes, I am planning to take a look at the new torchvision models this week and try to add them to my package.

**Comment by liminn** [comment]
@creafz  Great!

**Comment by creafz** [comment]
I have uploaded cnn_finetune 0.6.0 to PyPi, it contains new pretrained models from torchvision: `resnext50_32x4d`, `resnext101_32x8d`, `mobilenet_v2`, `shufflenet_v2_x0_5`, `shufflenet_v2_x1_0`, `googlenet`. Please open a new issue if you encounter any problem with the new models.
